posthog/session_recordings/queries/session_recording_list_from_filters.py

```diff
@@ -224,11 +238,19 @@ def _where_predicates(self) -> Union[ast.And, ast.Or]:
                 )
             )
 
-        remaining_properties = self._strip_person_and_event_properties(self._filter.property_groups)
-        if remaining_properties:
-            logger.info(
-                "session_replay_query_builder has unhandled properties", unhandled_properties=remaining_properties
+        cohort_subquery = CohortPropertyGroupsSubQuery(self._team, self._filter, self.ttl_days).get_queries()
+        if cohort_subquery:
+            optional_exprs.append(
+                ast.CompareOperation(
+                    op=ast.CompareOperationOp.In,
+                    left=ast.Field(chain=["s", "distinct_id"]),
+                    right=cohort_subquery,
+                )
             )
+
+        remaining_properties = self._strip_person_and_event_and_cohort_properties(self._filter.property_groups)
+        if remaining_properties:
+            posthoganalytics.capture_exception(UnexpectedQueryProperties(remaining_properties))
```

Comment:
i think anything in here is likely going to cause a join and impact performance, lets use exception capture to see when that happens